ground/gcs/src/plugins/setupwizard/pages/inputpage.ui

```diff
@@ -447,6 +447,48 @@ p, li { white-space: pre-wrap; }
        </property>
       </widget>
      </item>
+     <item>
+      <widget class="QToolButton" name="tbscrossfireButton">

```

Comment:
Yeah, SumH and SumD were using the lowercase variant.
Will align everything on your variant.